src/main/webapp/app/home/home.component.ts

```diff
@@ -105,6 +90,34 @@ export class HomeComponent implements OnInit, AfterViewChecked {
         this.registerAuthenticationSuccess();
     }
 
+    /**
+     * Initializes the component with the required information received from the server.
+     * @param profileInfo The information from the server how logins should be handled.
+     * @private
+     */
+    private initializeWithProfileInfo(profileInfo: ProfileInfo) {
+        this.profileInfo = profileInfo;
+
+        if (profileInfo.activeProfiles.includes('jira')) {
+            this.externalUserManagementUrl = profileInfo.externalUserManagementURL;
+            this.externalUserManagementName = profileInfo.externalUserManagementName;
+            if (profileInfo.allowedLdapUsernamePattern) {
+                this.usernameRegexPattern = new RegExp(profileInfo.allowedLdapUsernamePattern);
+            }
+        } else {
+            // TODO: in the future we might also allow external user management for non jira profiles
+            this.externalUserManagementActive = false;
+        }
+
+        this.accountName = profileInfo.accountName;
+        if (this.accountName === 'TUM') {
+            this.errorMessageUsername = 'home.errors.tumWarning';
+        }
+
+        this.isRegistrationEnabled = profileInfo.registrationEnabled || false;
+        this.needsToAcceptTerms = profileInfo.needsToAcceptTerms || false;
```

Comment:
Small sugestion to make the code shorter:
```suggestion
        this.isRegistrationEnabled = !!profileInfo.registrationEnabled;
        this.needsToAcceptTerms = !!profileInfo.needsToAcceptTerms;
```